app.rb

```diff
@@ -0,0 +1,101 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize()
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "[#{person.type}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_a_person(choice, name, age)
+    if choice == '1'
+      person = create_a_student(name, age)
```

Comment:
- [ ] I cannot create a student. Kindly fix this.

  ![image](https://github.com/waelelsafty07/oop-school/assets/36542986/276262f9-bc5a-41f6-82e5-a4a4293607d0)